projects/core/components/notification/notification.style.less

```diff
@@ -26,7 +31,8 @@
     }
 
     &[data-size='m'] {
-        padding: 0.75rem;
+        padding: 0.75rem 1rem;
```

Comment:
Changes to the padding don't seem right, now the icon on the left is too far from the edge, it should be equally distanced.
![image](https://github.com/taiga-family/taiga-ui/assets/11832552/d468cce6-53dd-440e-ad1d-47d44833811d)

Reply:
fixed